### PR TITLE
Create activateFabricBot.yml

### PR DESCRIPTION
**Pull request by astatide**
Hello! This PR is to convert the existing FabricBot users inside of the GitHub org to the new gitops-based FabricBot; while this file activates it org wide, it'll only act on repositories with an existing FabricBot configuration. Existing workflows should work the same, but the new FabricBot has a host of new features on a much more maintainable and scalable code base.

The plan is to prep this PR for merging and 'swap' (IE, deactivate the old FabricBot and merge) on February 1st. Please let me know if you have any questions/comments/etc!

### CODE REVIEW

**Review by zkoppert — APPROVED** [review]
👍🏻

File: policies/activateFabricBot.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: Fabric Bot
+description: Description
+
+resource: repository
+
```
Comment (Arekn0870) [inline policies/activateFabricBot.yml:5]: 5

**Review by Arekn0870 — APPROVED** [review]
5

**Comment by Arekn0870** [comment]
> Hello! This PR is to convert the existing FabricBot users inside of the GitHub org to the new gitops-based FabricBot; while this file activates it org wide, it'll only act on repositories with an existing FabricBot configuration. Existing workflows should work the same, but the new FabricBot has a host of new features on a much more maintainable and scalable code base.
> 
> The plan is to prep this PR for merging and 'swap' (IE, deactivate the old FabricBot and merge) on February 1st. Please let me know if you have any questions/comments/etc!

5

**Comment by Arekn0870** [comment]
> > Hello! This PR is to convert the existing FabricBot users inside of the GitHub org to the new gitops-based FabricBot; while this file activates it org wide, it'll only act on repositories with an existing FabricBot configuration. Existing workflows should work the same, but the new FabricBot has a host of new features on a much more maintainable and scalable code base.
> > 
> > The plan is to prep this PR for merging and 'swap' (IE, deactivate the old FabricBot and merge) on February 1st. Please let me know if you have any questions/comments/etc!
> 
> 5

**Comment by Dawidowski00** [comment]
Hi

**Review by olddogjosh — APPROVED** [review]

**Comment by Hektoreth** [comment]
Thank you for help ... Now my contract is aproved? Sory my eng is No.good

**Review by Thitipong0 — COMMENTED** [review]
no no

**Review by Thitipong0 — COMMENTED** [review]
ไม่ให้เปรียน

**Pull request closed without merging**